aidial_adapter_vertexai/embedding/multi_modal.py

```diff
@@ -191,7 +197,7 @@ async def embeddings(
 
         # NOTE: The model doesn't support batched inputs
         tasks: List[Callable[[], Tuple[Embedding, int]]] = []
-        async for sub_request in await get_requests(request, self.storage):
+        async for sub_request in await get_requests(self.storage, request):
```

Comment:
```python
async def get_requests(
    request: EmbeddingsRequest,
    storage: FileStorage | None,
) -> AsyncIterator[ModelRequest]:
```

Order of arguments is wrong

Reply:
I swapped the arguments consistently in `get_requests`.
Not sure why you see the old version of `get_requests`...